File: articles/from-baghdad-to-latin-europe-algorism-algebra-and-reception/fortran/place_value.f90
program place_value_example
  implicit none
  integer :: digits(4), value, i
  digits = (/1, 2, 3, 0/)
  value = 0
  do i = 1, size(digits)
    value = value * 10 + digits(i)
  end do
  print *, "place_value=", value
end program place_value_example

File: articles/from-baghdad-to-latin-europe-algorism-algebra-and-reception/java/PlaceValue.java
public class PlaceValue {
    public static void main(String[] args) {
        int[] digits = {1, 2, 3, 0};
        int value = 0;
        for (int digit : digits) value = value * 10 + digit;
        System.out.printf("place_value=%d%n", value);
    }
}
